Resources/Prototypes/InventoryTemplates/diona_inventory_template.yml

```diff
@@ -0,0 +1,104 @@
+﻿#human but no shoes
```

Comment:
If they're supposed to be made of wood maybe you should give them the "ShoesRequiredStepTriggerImmune" tag so they aren't griefed by broken glass